controllers/mdutil/util.go

```diff
@@ -44,17 +44,24 @@ const (
 	DefaultMachineDeploymentUniqueLabelKey = "machine-template-hash"
 
 	// FailedMSCreateReason is added in a machine deployment when it cannot create a new machine set.
+	// Deprecated
```

Comment:
```suggestion
	// Deprecated: This field will be removed in a next release.
```
nit

Looks like we usually write either use another func or this will be dropped.